### PR TITLE
Added pd deployer option

### PR DESCRIPTION
**Pull request by jjennings-va**
https://vasdvp.atlassian.net/browse/APD-43

### CODE REVIEW

File: products/provider-directory.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DU_ARTIFACT=health-apis-provider-directory-deployment
+DU_VERSION=1.0.01
+DU_NAMESPACE=pd
```
Comment (jblefkove-va) [inline products/provider-directory.conf]: namespace is `provider-directory`

File: products/provider-directory.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+              servicePort: 8082
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: pd-resource-quota
+  namespace: $DU_NAMESPACE
+spec:
+  hard:
+    limits.cpu: "6000m"
```
Comment (bschofield-va) [inline products/provider-directory.yaml]: What are we basing these number on?

Comment (jblefkove-va) [reply products/provider-directory.yaml]: Placeholders based on what CCE uses until we have more concrete numbers

**Comment by jblefkove-va** [comment]
This is ready

File: products/provider-directory.yaml (hunk 2 above)
Comment (mramirez-va) [reply products/provider-directory.yaml]: LGTM

**Pull request merged**